mgen/random_rotation.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+def random_rotation(n):
+    '''
+    Generate a nxn rotation matrix with a random direction in n-dimensional space
+    and angle of rotation.
+    The distribution of rotations is uniform on the n-sphere.
+    
+    :param n: dimension of space in which the rotation operates
+	:type n: int
+	:returns: rotation matrix
+	:rtype: a nxn :any:`numpy.ndarray`
+	'''
+    if n < 2:
+        raise ValueError('n must be greater than 1 but is ' + n + '.')
```

Comment:
`'text' + 1` does not work in python. Try `'text {} and a number in between'.format(1)`.